go/pkg/filemetadata/filemetadata_test.go

```diff
@@ -100,15 +103,17 @@ func TestComputeFilesWithXattr(t *testing.T) {
 			after := time.Now().Truncate(time.Second).Add(time.Second)
 			defer os.RemoveAll(filename)
 			got := Compute(filename)
+			wantDigest, wantErr := digest.NewFromString(fmt.Sprintf("%s/%d", tc.name, len(tc.contents)))
 			if got.Err != nil {
-				t.Errorf("Compute(%v) failed. Got error: %v", filename, got.Err)
+				if diff := cmp.Diff(wantErr.Error(), got.Err.Error()); diff != "" {
+					t.Errorf("Compute(%v) returned diff Err. (-want +got)\n%s", filename, diff)
+				}
 			}
-			wantDigest, _ := digest.NewFromString(fmt.Sprintf("%s/%d", tc.name, len(tc.contents)))
```

Comment:
This wantErr was not captured before, I am asserting it at line 106 on the right hand side now.